src/lib/plugins/system-info.js

```diff
@@ -0,0 +1,517 @@
+// This library contains several chunks of code from Sebastian Hildebrandt's
+// systeminformation library, MIT
+// https://github.com/sebhildebrandt/systeminformation
```

Comment:
Ill make the directory change for the file and update the PR. Check the systeminformation library and if it works and you prefer to use that one we can do it instead of this.